### PR TITLE
PR1

### LINKED ISSUE #2: test
test

### PR DESCRIPTION
**Pull request by Kexplx**

### CODE REVIEW

**Comment by thomasRiecher93** [comment]
test comment 1
test comment 2

File: file-1 [hunk 1]
```diff
@@ -1 +1,2 @@
 t-1
+t-2
```
Comment (thomasRiecher93) [inline file-1:8]: What is this?

```typescript
console.log("Hello World");
function tet(): void {
    console.log('Returning nothing');
}
```

**Comment by Kexplx** [comment]
> test comment 1

citing comment test

File: file-1 (hunk 1 above)
Comment (Kexplx) [reply file-1:8]: ewf

**Pull request closed without merging**